internal/restic/cache.go

```diff
@@ -1,37 +0,0 @@
-package restic
-
-import "io"
-
-// Cache manages a local cache.
-type Cache interface {
-	// BaseDir returns the base directory of the cache.
-	BaseDir() string
-
-	// Wrap returns a backend with a cache.
-	Wrap(Backend) Backend
-
-	// IsNotExist returns true if the error was caused by a non-existing file.
-	IsNotExist(err error) bool
-
-	// Load returns a reader that yields the contents of the file with the
-	// given id if it is cached. rd must be closed after use. If an error is
-	// returned, the ReadCloser is nil. The files are still encrypted
-	Load(h Handle, length int, offset int64) (io.ReadCloser, error)
```

Comment:
Please lower-case `Load`, `SaveWriter` and `Remove` in `internal/restic/file.go` to private methods. These aren't used anywhere outside the cache

Reply:
Done.